gen3/bin/kube-setup-wts.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Deploy workspace-token-service into existing commons,
+# this is an optional service that's not part of gen3 core services
+
+_KUBE_SETUP_WTS=$(dirname "${BASH_SOURCE:-$0}")  # $0 supports zsh
+source "${_KUBE_SETUP_WTS}/../lib/kube-setup-init.sh"
+
+
+echo "check wts secret"
+if ! g3kubectl get secret wts-secret > /dev/null 2>&1; then
+    hostname=$(g3kubectl get configmaps/global -o=jsonpath='{.data.hostname}')
```

Comment:
you can save the secret to a file under "$(gen3_secrets_folder)/g3auto/wts/", and if the k8s secret gets delete, then `gen3 secrets sync` will include the saved file as an entry in `wts-g3auto` secret, so something like this:
```
if k8s secret does not exist; then
  if file does not exist; then
     create the client and file
  fi
  gen3 secrets sync
fi
```